PlottingAnalysis/TS_TopFeatures.m

```diff
@@ -174,7 +198,7 @@
 fprintf(1,'Comparing the (in-sample) performance of %u operations for %u classes using a %s...\n',...
                                 height(Operations),numClasses,cfnName);
 timer = tic;
-testStat = giveMeStats(TS_DataMat,TimeSeries.Group,true);
+[testStat,CVMdl] = giveMeStats(TS_DataMat,TimeSeries.Group,true);
```

Comment:
1. for most cases this will not be a model containing the folds for CV.
2. This variable, `CVMdl` is redundant (not used again in this function?)

Reply:
True that it's not used. I've removed it now